include/deal.II/grid/manifold.h

```diff
@@ -201,6 +334,45 @@ class Manifold : public Subscriptor
    */
   Point<spacedim>
   get_new_point_on_cell (const typename Triangulation<dim,spacedim>::cell_iterator &cell) const;
+
+  /// @}
+
+  /**
+   * @name Computing direction vectors
+   */
+  /// @{
+
+  /**
+   * Return a "direction" vector that, at $\mathbf x_1$, is tangential to
```

Comment:
Since we renamed it, maybe change the title to "Computing tangent vectors", and here remote "direction", i.e.,  

`Return a vector that,  at $\mathbf x_1$, is tangential to`